{
    "type": "bundle",
    "id": "bundle--f1611327-70de-4cb8-8f91-fd116cf1674b",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--ac7b9e49-c1c4-46ec-9da1-9fdcda3858f1",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-04-17T01:54:56.072303Z",
            "modified": "2025-04-17T01:54:56.072303Z",
            "name": "CVE-2025-22063",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\nnetlabel: Fix NULL pointer exception caused by CALIPSO on IPv4 sockets\n\nWhen calling netlbl_conn_setattr(), addr->sa_family is used\nto determine the function behavior. If sk is an IPv4 socket,\nbut the connect function is called with an IPv6 address,\nthe function calipso_sock_setattr() is triggered.\nInside this function, the following code is executed:\n\nsk_fullsock(__sk) ? inet_sk(__sk)->pinet6 : NULL;\n\nSince sk is an IPv4 socket, pinet6 is NULL, leading to a\nnull pointer dereference.\n\nThis patch fixes the issue by checking if inet6_sk(sk)\nreturns a NULL pointer before accessing pinet6.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-22063"
                }
            ]
        }
    ]
}